test/e2e/vre-ch31-helpers.js

```diff
@@ -0,0 +1,108 @@
+const mock = require('./mock-helpers');
+
+function initApplicationSubmitMock() {
+  mock(null, {
+    path: 'v0/vre/okay',
+    verb: 'post',
+    value: {
+      data: {
+        attributes: {
+          guid: '123fake-submission-id-567'
+        }
+      }
+    }
+  });
+}
+
+function initDocumentUploadMock() {
+  mock(null, {
+    path: '/v0/vic/supporting_documentation_attachments',
+    verb: 'post',
+    value: {
+      data: {
+        attributes: {
+          guid: '123fake-submission-id-567'
+        }
+      }
+    }
+  });
+}
+
+function completeContactInformation(client, data) {
+  client
+    .fill('input[name="root_daytimePhone"]', data.daytimePhone)
+    .fill('input[name="root_eveningPhone"]', data.eveningPhone)
+    .fill('input[name="root_email"]', data.email)
+    .fill('input[name="root_view:confirmEmail"]', data.email);
+}
+
+function completeDisabilityInformation(client, data) {
+
+  client
+    .selectDropdown('root_disabilityRating', data.disabilityRating)
+    .fill('input[name=root_disabilities]', data.disabilities)
+    .fill('input[name=root_vaRecordsOffice]', data.vaRecordsOffice)
+    .selectRadio('root_view:inHospital', data['view:inHospital'])
+    .fill('input[name=root_view\\:hospital_hospitalName]', data.hospitalName)
+    .fillAddress('root_view:hospital_hospitalAddress', data.hospitalAddress);
+}
+
+function completeDischargeDocumentUpload(client) {
+  client
+    .setValue('input#root_dischargeDocuments', require('path').resolve(`${__dirname}/VA40-10007.pdf`));
```

Comment:
I copied this from vic-v2 but I don't think it's actually doing anything.

Reply:
The document upload e2e stuff doesn't work very well on Jenkins/Electron, so probably not.